sequence.go

```diff
@@ -92,3 +92,21 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+//AnonymizeSequence takes a sequence struct and substitute identification data for a new anonymized string and returns this modified Sequence struct.
+func anonymizeSequence(sequence Sequence, sequence_id string) Sequence {
+
+        anonymizer := "anonymized"
+
+        if (sequence_id != "") {
+            anonymizer = sequence_id
+        }
+
+        sequence.Meta.Name = anonymizer
+        sequence.Meta.Keywords = ""
+        for i := range sequence.Features {
```

Comment:
Is this how easy_dna does it? I thought they would have just created a new record and inserted ID + sequence string.

Reply:
They [create a deepcopy](https://github.com/Edinburgh-Genome-Foundry/easy_dna/blob/14f694d549682bd0e2aeaa136cc82c0f20015c11/easy_dna/record_operations.py#L106) of the sequence and after that [assigns the ID for the anonymous value](https://github.com/Edinburgh-Genome-Foundry/easy_dna/blob/14f694d549682bd0e2aeaa136cc82c0f20015c11/easy_dna/record_operations.py#L112). 

Do you think deepcopy it's a necessary step for this?

** The links are permalinks for the related lines of code in easy_dna **